.ci/scripts/docker-build.sh

```diff
@@ -3,8 +3,16 @@ set -eox pipefail
 
 TAG=${BRANCH_NAME:-'latest'}
 
+# Read node version from .nvmrc and remove v prefix.
+NODE_VERSION=$(cat .nvmrc | sed "s#v##g")
+
+# Download the docker image if possible
+docker pull docker.elastic.co/observability-ci/synthetics-recorder:$TAG || true
+
 docker \
   build \
   --file e2e/Dockerfile.jenkins \
   --build-arg DISPLAY=":99" \
+  --build-arg NODE_VERSION="${NODE_VERSION}" \
+  --build-arg UID="$UID" \
```

Comment:
Ensure the UID is populated to the `node` user so when mounting the volume the docker container can write in the directory. Docker volumes in MacOS and Linux differ a bit, so I could not reproduce the issue locally since I use MacOS but it failed in Linux unless resetting the UID for the node user

Reply:
Thanks for the info!!